src/components/order-service/order-service-info.html

```diff
@@ -31,7 +31,7 @@
       </span>
       <span ng-if="$ctrl.selectedPlan.spec.description">{{$ctrl.selectedPlan.spec.description}}</span>
     </p>
-    <p ng-if="$ctrl.description" ng-bind-html="$ctrl.description | linky : '_blank'" class="description"></p>
+    <p ng-bind-html="($ctrl.description | linky : '_blank') || 'No description provided.'" class="description"></p>
```

Comment:
I don't think we should show "No description provided." if there's a long description. Only if there's no description at all (short or long).